addons/project/views/project_views.xml

```diff
@@ -18,7 +18,7 @@
                <search string="Tasks">
                     <field name="name" string="Task"/>
                     <field name="tag_ids"/>
-                    <field name="user_ids" context="{'active_test': False}"/>
+                    <field name="user_ids"/>
```

Comment:
It is not a bug because when a task has an archived user assigned, we could find that task with that user.

Reply:
To manage the issue, I explain above, we should take into account the archived users too. 🙂 
```suggestion
                    <field name="user_ids" filter_domain="[('user_ids', 'ilike', self), ('user_ids.active', 'in', [True, False])]"/>
```